.github/workflows/main.yml

```diff
@@ -63,9 +63,17 @@ jobs:
       uses: actions/setup-dotnet@v4
       with:
         dotnet-version: ${{ env.DOTNET_VERSION }}
-        
-    - name: Test
-      if: matrix.os != 'ubuntu-latest'
+
+    - name: Test (MacOS)
+      if: matrix.os == 'macos-latest'
+      run: |
+        brew install leveldb
+        dotnet build
+        cp -vp /opt/homebrew/Cellar/leveldb/1.23_1/lib/libleveldb.dylib ./tests/Neo.Plugins.Storage.Tests/bin/Debug/net8.0/
```

Comment:
This instruction will fail with the next dotnet version upgrade. We'd better fetch dotnet version from environment to construct the destination path.

Reply:
We will have to update all these files anyways. If we upgrade plus dotnet 9.0 is LTS

Reply:
> We will have to update all these files anyways.

But it would be nice to make as less changes as possible during the update. Looks like a good practice to me.